src/components/manager/ManagerDropBoxContent.js

```diff
@@ -194,15 +187,11 @@ const FileDisplay = ({file, tree, treeLoading}) => {
                     </Document>
                 );
             } else if (fileExt === "json") {
-                return (
-                    <ReactJson
-                        src={JSON.parse(fileContents[file] || "{}")}
-                        displayDataTypes={false}
-                        enableClipboard={false}
-                        name={null}
-                        collapsed={true}
-                    />
-                );
+                if (Array.isArray(fileContents[file])) {
+                    return ( <JsonArrayDisplay doc={fileContents[file] || []} standalone/> );
```

Comment:
formatting: inconsistent spacing + needless parens with this line and the return below it